doc/sphinx/addendum/canonical-structures.rst

```diff
@@ -313,7 +313,9 @@ constructor ``*``. It also tests that they work as expected.
 Unfortunately, these declarations are very verbose. In the following
 subsection we show how to make them more compact.
 
-.. coqtop:: all
+.. FIXME shouldn't warn
```

Comment:
@vbgl Have you figured out the meaning of the canonical structure warnings?

Reply:
No sorry.